ci/build.yml

```diff
@@ -185,10 +185,10 @@ crypto build:
 legacy fw regular build:
   stage: build
   needs: []
-  variables:
-    MEMORY_PROTECT: "0"
   script:
-    - nix-shell --run "poetry run legacy/script/cibuild"
+    - nix-shell --run "export MEMORY_PROTECT=1 && poetry run legacy/script/cibuild"
+    - nix-shell --run "make -C legacy clean"
```

Comment:
`make clean` is not enough.
that should _probably_ be fixed, but for now you should run `legacy/script/setup`
dtto below

Reply:
Fixed in 81e9d28.